03/app.js

```diff
@@ -41,21 +41,28 @@ books.getAuthor = function (isbn) {
 };
 
 books.getTitle = function (isbn, lang) {
-	const language = lang;
-	const isTitleAvailable = this[isbn]['title'][language];
-	if (isTitleAvailable) {
-		return isTitleAvailable;
+	if (isbn) {
```

Comment:
Bardziej chodziło mi o `this[isbn]` ale najlepiej będzie zrobić tak: `if(isbn && this[isbn])`